lib/utils.coffee

```diff
@@ -293,7 +295,7 @@ exports.requestAsync = (options, retriesRemaining) ->
 	delete opts.timeout
 
 	requestTime = new Date()
-	p = fetch(url, opts)
+	p = exports.fetch(url, opts)
```

Comment:
Ok, it makes sense, but I've never seen this pattern before (the internal side - I've seen external usage), it might be worth making sure there is no reference to `fetch` available and adding a comment so that it's impossible to slip up and use just `fetch`